src/Microsoft.TestPlatform.CommunicationUtilities/Interfaces/IDataCollectionTestCaseEventHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.Interfaces
+{
+    /// <summary>
+    /// Interface for interacting with execution process for getting test case events in datacollection process.
+    /// </summary>
+    internal interface IDataCollectionTestCaseEventHandler
```

Comment:
We are creating a lot of handler interfaces. Can we rename this to IRequestHandler and then extend this interface to create ITestRequesHandler for example. Just a suggestion.